apps/ccd/ccd-definition-store-api/demo-image-policy.yaml

```diff
@@ -6,8 +6,7 @@ metadata:
     hmcts.github.com/prod-automated: disabled
 spec:
   filterTags:
-    pattern: '^pr-1474-[a-f0-9]+-(?P<ts>[0-9]+)'
-    extract: '$ts'
```

Comment:
We should keep this line.

Reply:
extract: '$ts', right? It wasn't there when I made this change, but if this line works with prod, it will make it easier for us later.